SUBMISSIONS.md

```diff
@@ -0,0 +1,177 @@
+|[![](https://avatars.githubusercontent.com/u/7424983?s=30)](https://github.com/Blockstream/miniscript-templates)|[000](mint-000.md)|[001](mint-001.md)|[002](mint-002.md)|
+|----------|----------|----------|----------|
+
+<H1>
+
+<center>Submission Format</center>
```

Comment:
In 97142059eea7312d73ee8c710b31db2a1a94318a:

It looks like SUBMISSIONS.md is a grab-bag of markdown tests and experiments. Maybe it should be removed?

Reply:
resolved in: 5b7a4e5fcbae45e5e1fce0106ed799e43bea6f0d